Build error for check

Attempted to build on Linux Mint:
Error:
../src/protocol/.libs/libsub_protocol.a(request_list.o): In function `torrent::RequestList::delegate()':
/home/rileymat/Development/libtorrent/src/protocol/request_list.cc:127: undefined reference to`torrent::Delegator::delegate(torrent::PeerChunks*, int)'
collect2: error: ld returned 1 exit status

This is the same error that issue #51 had.

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -10,9 +10,9 @@ LibTorrentTest_LDADD = \
 	../src/torrent/peer/libsub_torrentpeer.la \
 	../src/data/libsub_data.la \
 	../src/dht/libsub_dht.la \
-	../src/download/libsub_download.la \
 	../src/net/libsub_net.la \
 	../src/protocol/libsub_protocol.la \
+	../src/download/libsub_download.la \
 	../src/tracker/libsub_tracker.la \
 	../src/utils/libsub_utils.la \
 	../src/torrent/utils/libsub_torrentutils.la

